src/components/auth/Register/index.jsx

```diff
@@ -5,7 +5,7 @@ import { Input } from '@/components/ui/input'
 import { Button } from '@/components/ui/button'
 import { Label } from '@/components/ui/label'
 import { registerWithEmailAndPassword, signInWithGoogle } from '@/firebase/auth'
-import { createUserDocument } from '@/firebase/firestore'
+import { createUserDocument } from '../../../utils/createUserCollection'

```

Comment:
dont we have a @/utils ?

Reply:
ı have tried to edit it. not sure why it is not working. having warning when ı use @utils.

Reply:
<img width="382" alt="image" src="https://github.com/vaionex/VBS/assets/24817172/b2129675-a42f-4cd3-bcec-a9ea2e15c374">
you have to add it in the jsconfig.json in the root of the project